packages/fx-core/resource/package.nls.json

```diff
@@ -852,6 +852,7 @@
   "driver.apiKey.error.clientSecretInvalid": "Client secret is invalid. The length of client secret should be in this range: >=10 and <=128",
```

Comment:
Proposed: Invalid client secret. It should be 10 to 128 characters long.

Reply:
Updated, thanks!